### DISCUSSION TITLE
Github gave me a faculty account.

### DISCUSSION THREAD

**Discussion by rvsahoo** [community/community#39450] [GitHub Education]
This is urgent please help!!!!!

I applied for a student developer pack through my institution email. Although my account has been verified, it is now a faculty account with classroom benefits. How can I change it to a students' account to avail benefits of the Student Developer Pack?? 
My other colleagues are experiencing the same issue.

**Comment by dhrubasaha08** [comment]
@rvsahoo Are you a student or a professor?

**Reply by rvsahoo** [reply]
I am a student.

**Reply by rvsahoo** [reply]
During the application process I had submitted my student ID card.

**Reply by dhrubasaha08** [reply]
Try to reverify your status using a different mail id along with a proof that you're a student!

**Comment by airtower-luna** [comment]
Try to [contact GitHub support](https://support.github.com/request) about that, us other users can't help there. Good luck!
